plugin.json

```diff
@@ -0,0 +1,88 @@
+{
+    "id": "jitsi",
+    "name": "Jitsi",
+    "description": "Jitsi audio and video conferencing plugin for Mattermost.",
+    "homepage_url": "https://github.com/mattermost/mattermost-plugin-jitsi",
+    "support_url": "https://github.com/mattermost/mattermost-plugin-jitsi/issues",
+    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-jitsi/releases/tag/v1.3.0",
+    "icon_path": "assets/icon.svg",
+    "version": "1.3.0",
+    "min_server_version": "5.2.0",
+    "server": {
+        "executables": {
+            "linux-amd64": "server/dist/plugin-linux-amd64",
+            "darwin-amd64": "server/dist/plugin-darwin-amd64",
+            "windows-amd64": "server/dist/plugin-windows-amd64.exe"
+        }
+    },
+    "webapp": {
+        "bundle_path": "webapp/dist/main.js"
+    },
+    "settings_schema": {
+        "settings": [
+            {
+                "key": "JitsiURL",
+                "display_name": "Jitsi Server URL",
+                "type": "text",
+                "help_text": "The URL for an on-premise Jitsi server. For example, https://jitsi.example.com.",
+                "placeholder": "https://jitsi.example.com"
+            },
+            {
+                "key": "JitsiEmbedded",
+                "display_name": "Embed Jitsi video inside Mattermost",
+                "type": "bool",
+                "help_text": "(Experimental) When true, Jitsi video is embedded as a floating window inside Mattermost."
+            },
+            {
+                "key": "JitsiJWT",
```

Comment:
There are other ways to identifiy yourself with jitsi, but jitsi is a pretty complex ecosystem of diferent apps, so adding another authentication methods probably involve setting up prosody plugins, this is pretty straight forward and I think is pretty universal in the jitsi installations. I can't say the same about OAuth.

Reply:
A bit of context here, we had problems with Zoom plugin and JWT because the casing was different between what was registered on Mattermost and what was registered on Zoom. Maybe Jitsi handles this more elegantly, but it would be great if we had more authentication systems.

Reply:
I think is pretty save the JWT in that sense, actually we don't initially create accounts in jitsi itself, the accounts access right away and are identified by the user id, no by name or other fields.

As I said, we can try to find other authentication methods, but I would wait until is explicitly requested, because I think this solution covers a huge spectrum of the use cases.